Applications/Games/Overwatch/Online/script.js

```diff
@@ -10,19 +10,20 @@ var installerImplementation = {
             .name("Overwatch")
             .editor("Blizzard")
             .applicationHomepage("http://www.playoverwatch.com/")
-            .author("ImperatorS79")
+            .author("ImperatorS79, kreyren")
             .url("https://eu.battle.net/download/getInstaller?os=win&installer=Overwatch-Setup.exe")
         //The checksum is different each time you download
             .wineVersion(LATEST_STAGING_VERSION)
-            .wineDistribution("staging")
-            .wineArchitecture("amd64")
+            .wineDistribution("staging") // ESYNC required
+            .wineArchitecture("amd64") // Multilib preffered? (without setting wine architecture)
             .category("Games")
             .executable("Overwatch.exe")
             .preInstall(function (wine/*, wizard*/) {
-                wine.windowsVersion("winxp");
+                wine.windowsVersion("win7");
                 wine.vcrun2015();
                 wine.corefonts();
                 wine.enableCSMT();
```

Comment:
more info? Windows 7 WINE environment and higger is mandatory

Reply:
I speak about the csmt line.

Reply:
And win7 is normally the default, so no need to specify it.

Reply:
@ImperatorS79 It's probably sane to specify win7 environment it anyway in case there is a change in wine or something alike..

didn't test CSMT, not sure if needed

Reply:
In fact, the experience proved us that you have more chance to have your script broken if you specify unneeded features. 
Even if there is a change in wine, win10 setting is very likely to be compatible.
However, there can also be a change in battle.net app

Reply:
Based on my experience seting w10 might cause issues in some wineapps. Recommends specifying `win7`. Will fix if it causes issues.